### PR TITLE
Bug fix for graphite walk array options

### PR DESCRIPTION
**Pull request by mrh666**
Bug fix for graphite walk array options - typos fixed

### CODE REVIEW

File: templates/influxdb.conf.erb [hunk 1]
```diff
@@ -125,8 +125,8 @@ reporting-disabled = <%= @reporting_disabled %>
 <% elsif @graphite_options.is_a?(Array) -%>
 <% @graphite_options.each do |option| -%>
 [[graphite]]
-<% @options.key.sort.each do |key| -%>
-<% val = @options[key] 
+<% option.keys.sort.each do |key| -%>
```
Comment (n1tr0g) [inline templates/influxdb.conf.erb]: I think you forgot the @ at the start of the variable name.

Comment (mrh666) [inline templates/influxdb.conf.erb]: Nope, it just doesn't work with @

**Comment by n1tr0g** [comment]
super interesting ... what version of puppet you use ?

**Comment by mrh666** [comment]
@n1tr0g 3.8.7

**Comment by n1tr0g** [comment]
ok let me try and recreate the issue. In the meantime i will release the current new version as it is ... Once i verify that I will release another minor release. tnx again for the patience.

**Comment by mrh666** [comment]
@n1tr0g Thank you for maintaining this package!

**Comment by mrh666** [comment]
@n1tr0g , I added array handler for tags in graphite (it was missed) please take a look.

**Comment by mrh666** [comment]
@n1tr0g Can you check this pull request please? Without it graphite functionality doesn't work.

**Comment by n1tr0g** [comment]
sorry about the delay. I will have a look tomorrow morning when I wake up and potentially release the new version the same day.

**Comment by mrh666** [comment]
I created a new pull request for the latest version.

**Pull request closed without merging**